src/main/java/me/zeroeightsix/kami/module/modules/chat/AntiSpam.kt

```diff
@@ -32,37 +33,46 @@ import java.util.stream.Collectors
+            .withValue(ReplaceMode.ASTERISKS).withVisibility { mode.value == Mode.REPLACE }.build())
+    private val p = register(Settings.e<Page>("Page", Page.TYPE))
 
     /* Page One */
-    private val discordLinks = register(Settings.booleanBuilder("DiscordLinks").withValue(true).withVisibility { p.value == Page.ONE }.build())
-    private val announcers = register(Settings.booleanBuilder("Announcers").withValue(true).withVisibility { p.value == Page.ONE }.build())
-    private val spammers = register(Settings.booleanBuilder("Spammers").withValue(true).withVisibility { p.value == Page.ONE }.build())
-    private val insulters = register(Settings.booleanBuilder("Insulters").withValue(true).withVisibility { p.value == Page.ONE }.build())
-    private val greeters = register(Settings.booleanBuilder("Greeters").withValue(true).withVisibility { p.value == Page.ONE }.build())
-    private val ips = register(Settings.booleanBuilder("ServerIps").withValue(true).withVisibility { p.value == Page.ONE }.build())
-    private val specialCharEnding = register(Settings.booleanBuilder("SpecialEnding").withValue(true).withVisibility { p.value == Page.ONE }.build())
-    private val specialCharBegin = register(Settings.booleanBuilder("SpecialBegin").withValue(true).withVisibility { p.value == Page.ONE }.build())
-    private val iJustThanksTo = register(Settings.booleanBuilder("IJustThanksTo").withValue(true).withVisibility { p.value == Page.ONE }.build())
+    private val discordLinks = register(Settings.booleanBuilder("Discord").withValue(true).withVisibility { p.value == Page.TYPE }.build())
+    private val slurs = register(Settings.booleanBuilder("Slurs").withValue(true).withVisibility { p.value == Page.TYPE }.build())
+    private val swears = register(Settings.booleanBuilder("Swears").withValue(false).withVisibility { p.value == Page.TYPE }.build())
+    private val automated = register(Settings.booleanBuilder("Automated").withValue(true).withVisibility { p.value == Page.TYPE }.build())
+    private val ips = register(Settings.booleanBuilder("ServerIps").withValue(true).withVisibility { p.value == Page.TYPE }.build())
+    private val specialCharEnding = register(Settings.booleanBuilder("SpecialEnding").withValue(true).withVisibility { p.value == Page.TYPE }.build())
+    private val specialCharBegin = register(Settings.booleanBuilder("SpecialBegin").withValue(true).withVisibility { p.value == Page.TYPE }.build())
+    private val greenText = register(Settings.booleanBuilder("GreenText").withValue(false).withVisibility { p.value == Page.TYPE }.build())
+    private val experimental = register(Settings.booleanBuilder("Experimental").withValue(false).withVisibility { p.value == Page.TYPE }.build())
+    private val fancyChat = register(Settings.booleanBuilder("FancyChat").withValue(false).withVisibility { experimental.value && p.value == Page.TYPE }.build())
```

Comment:
Just get rid of the experimental setting entirely

Reply:
will do!